core/chains/evm/config/toml/defaults.go

```diff
@@ -15,34 +18,40 @@ import (
 )
 
 var (
+	CUSTOM_DEFAULTS_ENV_KEY = "CL_CHAIN_DEFAULTS"
```

Comment:
Could leverage the `package env` for this

Reply:
nice, thanks